### PR TITLE
Common utils

### PR DESCRIPTION
**Pull request by HuifangYeo**
Update notebook to use common function in notebook_utils.ipynb

### CODE REVIEW

**Comment by HuifangYeo** [comment]
Managed to achieve the requested using 
```
import sys  

sys.path.insert(0, '../../utils')
import notebook_utils

cwd = os.getcwd()
# use function from notebook_utils
notebook_utils.download_source(cwd, 'http://data.atoti.io/notebooks/collateral-shortfall-monitoring/assets_positions.csv', "assets_positions.csv")
```

**Comment by HuifangYeo** [comment]
- Refactored to use append instead of delete.
- use cwd in download_source function instead of main.ipynb

```
import sys  

sys.path.append('../../utils')
import notebook_utils

# use function from notebook_utils
notebook_utils.download_source('http://data.atoti.io/notebooks/collateral-shortfall-monitoring/assets_positions.csv', "assets_positions.csv")
```
Refer to https://github.com/atoti/notebooks/pull/15

File: utils/notebook_utils.py [hunk 1]
```diff
@@ -2,14 +2,15 @@
 import os.path
 
 # check if file has already been downloaded 
-def download_source(cwd: str, url: str, filename: str):
+def download_source(url: str, filename: str):
     """This function downloads data file from the given url to the working directory (cwd).
 
     Args:
         cwd: current working directory.
```
Comment (jbe456) [inline utils/notebook_utils.py:9]: you should remove this line

File: utils/notebook_utils.py [hunk 2]
```diff
@@ -2,14 +2,15 @@
 import os.path
 
 # check if file has already been downloaded 
-def download_source(cwd: str, url: str, filename: str):
+def download_source(url: str, filename: str):
```
Comment (jbe456) [inline utils/notebook_utils.py:5]: can you remove "filename" as well from the argument? instead you could do
```python
filename = url.split('/')[-1]
```

**Comment by jbe456** [comment]
Closes #15

**Pull request merged**